web/modules/custom/speech/js/conference_call.js

```diff
@@ -0,0 +1,145 @@
+(function ($, Drupal) {
+
+    'use strict';
+
+    /**
+     * @type {Drupal~behavior}
+     */
+    Drupal.behaviors.conferenceCall = {
+        attach: function (context, settings) {
+            var callClient;
+            var call;
+
+            // Hides the answer and hangup buttons.
+            hideInput('answer');
+            hideInput('hangup');
+            $('audio#calling').trigger("pause");
+            $('audio#ringtone').trigger("pause");
+
+            // Defines the Sinch client.
+            var sinchClient = new SinchClient({
+                applicationKey: "1ece0175-849e-42d7-9b4b-b9fd1a1c1358",
```

Comment:
The application ID should be configurable in the BO and shouldn't be pushed in a public repository.